app.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { getTopics } = require('./controllers.js');
+const app = express();
+app.use(express.json());
+
+app.get('/api/topics', getTopics);
+
+require('./errors');
```

Comment:
What is this require doing?

Reply:
refactored as stephen explained today in the lecture.